nifi-toolkit/nifi-toolkit-tls/src/main/java/org/apache/nifi/toolkit/tls/standalone/TlsToolkitStandaloneCommandLine.java

```diff
@@ -168,6 +188,17 @@ protected CommandLine doParse(String... args) throws CommandLineParseException {
         clientPasswordsGenerated = commandLine.getOptionValues(CLIENT_CERT_PASSWORD_ARG) == null;
         overwrite = commandLine.hasOption(OVERWRITE_ARG);
 
+        if(commandLine.hasOption(SPLIT_KEYSTORE_ARG)) {
+            if(commandLine.hasOption(KEY_STORE_PASSWORD_ARG) && commandLine.hasOption(KEY_PASSWORD_ARG)) {
```

Comment:
I think `KEY_PASSWORD_ARG` can legitimately be empty if the keystore password and key password are the same (i.e. no explicit key password was set).

Reply:
Updated to use the keystore password if no key password is provided and added unit test.